CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Filenames of external files can no longer contain curly braces. [#5926](https://github.com/JabRef/jabref/pull/5926)
 - We made the filters more easily accessible in the integrity check dialog. [#5955](https://github.com/JabRef/jabref/pull/5955)
 - We reimplemented and improved the dialog "Customize entry types" [#4719](https://github.com/JabRef/jabref/issues/4719)
+- We added an APS fetcher. [#818](https://github.com/JabRef/jabref/issues/818)
```

Comment:
Can you also link APS please? I think, this is not a common term and it would help readers to get what it is. Maybe also write the long form of APS and add a link ^^.

Reply:
Done